components/PerangkatDesa/index.vue

```diff
@@ -146,6 +232,100 @@ export default {
     previousPage (value) {
       this.query.current_page = value
     },
+    verifyUser (item) {
+      this.dataPerangkatDesa = item
+      const { name } = item
+      this.contentPerangkatDesa.data = name
+      this.contentPerangkatDesa.header = 'Verifikasi Perangkat Desa'
+      this.contentPerangkatDesa.title = 'Apakah anda yakin ingin menerima Perangkat Desa ini?'
+      this.showDialogVerify = true
+    },
+    onRejectData () {
+      const { email } = this.dataPerangkatDesa
+      this.contentPerangkatDesa.data = email
+      this.contentPerangkatDesa.header = 'Email Penolakan'
+      this.contentPerangkatDesa.title = 'Dikirim kepada'
+      this.contentPerangkatDesa.showNotes = true
+    },
+    async onVerifyReject () {
+      const { id } = this.dataPerangkatDesa
+      try {
+        const response = await this.$axios.put(`/users/${id}/verify`, { is_verify: false, notes: this.contentPerangkatDesa.notes })
+        if (response) {
+          this.onClose()
+          this.$store.dispatch('dialog/showDialog', {
+            header: 'Penolakan Perangkat Desa Berhasil',
+            title: 'Penolakan Perangkat Desa telah berhasil dilakukan.',
+            message: this.contentPerangkatDesa.data,
+            detailMessage: 'Email terkait konfirmasi penolakan telah dikirimkan ke email Perangkat Desa bersangkutan.',
+            iconMessage: 'check-mark-circle',
+            iconColor: 'text-green-700',
+            btnLeftVariant: 'primary',
+            btnLeftLabel: 'Saya mengerti',
+            dialogType: 'information',
+            actionBtnLeft: () => this.$fetch()
+          })
+        }
+      } catch (error) {
+        const { response: { status, data: { errors } } } = error
```

Comment:
discussion:

Hello @yoslie , if there is an error when using Axios it will always return an error object by default so this line:
`const { response: { status, data: { errors } } } = error` will works just fine. 

Otherwise please be careful when destructuring nested objects from a object we can't control.

example:
<img width="660" alt="Screen Shot 2022-09-13 at 15 45 38" src="https://user-images.githubusercontent.com/33661143/189841876-748d4c6e-75d1-4262-81d3-4d177cecef92.png">


ref: 
[Axios Handling Error](https://axios-http.com/docs/handling_errors)
[Destructuring Nested Objects
](https://medium.com/@pyrolistical/destructuring-nested-objects-9dabdd01a3b8)

Reply:
sure thanks for reminder 🙏